src/middlewared/middlewared/plugins/device_/vrrp_events.py

```diff
@@ -13,13 +199,20 @@ def __init__(self, *args, **kwargs):
         super(VrrpFifoThread, self).__init__()
         self._retry_timeout = 2  # timeout in seconds before retrying to connect to FIFO
         self._vrrp_file = '/var/run/vrrpd.fifo'
+        self.pause_event = Event()
         self.middleware = kwargs.get('middleware')
-        self.logger = self.middleware.logger
-        self.shutdown_line = '--SHUTDOWN--\n'
+        self.event_queue = kwargs.get('queue')
```

Comment:
What is impact of `self.event_queue = None` (e.g. not in kwargs)?

Reply:
If it's mandatory, then we should have it be mandatory kwarg.

Reply:
agree, fixed